frontend_server/lib/server.dart

```diff
@@ -163,16 +179,18 @@ class _FrontendCompiler implements CompilerInterface {
 /// `compiler` is an optional parameter so it can be replaced with mocked
 /// version for testing.
 Future<int> starter(List<String> args, {CompilerInterface compiler,
-    Stream<List<int>> input, StringSink output}) async {
+    Stream<List<int>> input, StringSink output,
+    IncrementalKernelGenerator generator,
+    KernelSerializer kernelSerializer}) async {
```

Comment:
nit: style here too, similar as above (`{` at the end of the first line, then line break, if it contains any line breaks, and line break before the `}` if there's one after the `{`)